### ISSUE TITLE
The config cluster's east west gateway cannot connect the external cluster 

### ISSUE THREAD

**Issue by qianwens** [istio/istio#46090]
### Is this the right place to submit this?

- [X] This is not a security vulnerability or a crashing bug
- [X] This is not a question about how to use Istio

### Bug Description

I setup the external cluster, remote cluster and the secondary cluster in the tutorial: https://preliminary.istio.io/latest/docs/setup/install/external-controlplane/

But the remote cluster's eastwest gateway cannot connect the external server because the external server failed to get the kube client of the cluster.

Error in the external cluster's istio: Failed to authenticate client from 10.224.0.101:54272: Authenticator ClientCertAuthenticator: no verified chain is found; Authenticator KubeJWTAuthenticator: could not get cluster Kubernetes's kube client

I need some help to diagnostic why this error happens.

### Version

```Text
1.18.1
```

### Affected product area

- [ ] Ambient
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [ ] Upgrade
- [ ] Multi Cluster
- [ ] Virtual Machine
- [ ] Control Plane Revisions

**Comment by qianwens** [comment]
> Sounds like the client proxy is configured as clusterName=`Kubernetes`, but the control plane was configured with its cluster ID as a different name.
> 
> So probably when you installed you used inconsistent namings. Its hard to say more since you didn't provide any information about what you did.

Thanks for your quick response. I don't know why the clusterName=Kubernetes as I did not configure that name. I have two remote clusters, one cluster works well while another cluster which is also configured as the config cluster does not. So I made some changes here, use the external cluster as the config cluster instead of use the remote cluster. Now both remote cluster works well now. 

I don't know why it behaves like this as I am still new to Istio, I am wondering why the tutorial use the remote cluster as the config cluster, is it recommended to use the application workload cluster as the config cluster?
